backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/textfield/BpkTextField.kt

```diff
@@ -58,7 +63,55 @@ fun BpkTextField(
         modifier = modifier,
         readOnly = readOnly,
         placeholder = placeholder,
-        icon = icon,
+        prefix = {
+            if (icon != null) {
+                BpkIcon(
+                    icon = icon,
+                    contentDescription = null,
+                    size = BpkIconSize.Large,
+                    modifier = Modifier.padding(start = BpkSpacing.Sm),
+                    tint = animateColorAsState(
+                        when (status) {
+                            is BpkFieldStatus.Disabled -> BpkTheme.colors.textDisabled
+                            else -> BpkTheme.colors.textSecondary
+                        },
+                    ).value,
+                )
+            }
+        },
+        status = status,
+        visualTransformation = visualTransformation,
+        keyboardOptions = keyboardOptions,
+        keyboardActions = keyboardActions,
+        maxLines = maxLines,
+        interactionSource = interactionSource,
+        clearAction = clearAction,
+    )
+}
+
+@Composable
+fun BpkTextField(
```

Comment:
can you make this internal please? this should not be exposed to consumers (or just use the impl function from the search input summary)